example/.mage/magefile.go

```diff
@@ -24,20 +36,16 @@ func init() {
 	)
 }
 
-func SuccessfulInit() {
-	println(`
-Mage-tools has been successfully initialized!
-
-To get started, have a look at the magefile.go in the .mage directory,
-and look at https://github.com/einride/mage-tools#readme to learn more
-`)
-}
-
 func All() {
 	mg.Deps(
 		mg.F(mgconvco.ConvcoCheck, "origin/master..HEAD"),
```

Comment:
should this be `master` or `main`...? :joy:

Reply:
> should this be `master` or `main`...? joy

:rofl: could this be figured out from command line? Rather then hardcoding it :P

Reply:
`git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'` seems to work

Reply:
@Edholm maybe worth adding as a mage-tools helper function?

Reply:
Yep thought so too. I'll see if I have time to do that today.

Created #83 for now.

Reply:
👌